cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -114,6 +121,8 @@ instance FromJSON PartialNodeConfiguration where
 
       -- Node parameters, not protocol-specific
       pncSocketPath' <- Last <$> v .:? "SocketPath"
+      pncDiffusionMode'
+        <- Last . Just . getDiffusionMode <$> v .:? "DiffusionMode"
```

Comment:
Wait! This made it non-optional. It still needs to be optional but use mempty as the default value.

Reply:
That's a type error!

Reply:
I fixed it.

Reply:
It ought to be:
```
Last . fmap getDiffusionMode <$> v .:? "DiffusionMode"
```

This way it's an optional value, with `mempty :: Last Nothing` if not specified.